wagtail/admin/views/generic/models.py

```diff
@@ -256,13 +256,29 @@ def get_buttons(column, instance, *args, **kwargs):
         )
 
     def _get_custom_column(self, field_name, column_class=Column, **kwargs):
-        label, attr = label_for_field(field_name, self.model, return_attr=True)
+        *relation, field = field_name.split(".")
```

Comment:
Can we split by `django.db.models.constants.LOOKUP_SEP` instead?

Reply:
Changed